tests/functional/adapter/test_retries_iceberg.py

```diff
@@ -101,7 +101,8 @@ class TestIcebergRetriesEnabled:
     @pytest.fixture(scope="class")
     def dbt_profile_target(self):
         profile = copy.deepcopy(base_dbt_profile)
-        profile["num_iceberg_retries"] = 1
+        # we setup the retries to the same number as the parallelism
+        profile["num_iceberg_retries"] = 10
```

Comment:
```suggestion
        profile["num_iceberg_retries"] = PARALLELISM
```
mb reuse constant will suit better here ?